README.md

```diff
@@ -72,4 +67,41 @@ It returns a `hash` with the response body.
 
 ## Versioning
 
-Kolekti follows the [semantic versioning](http://semver.org/) policy.
+Likeno follows the [semantic versioning](http://semver.org/) policy.
+
+### License
+
+Copyright (c) 2013-2016 The Author developers.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
```

Comment:
Sorry for that; I copied it from another repository from mezuro, tough I can't remember which one now.